core/drivers/pmd.cc

```diff
@@ -207,6 +207,8 @@ CommandResponse PMDPort::Init(const bess::pb::PMDPortArg &arg) {
 
   int num_txq = num_queues[PACKET_DIR_OUT];
   int num_rxq = num_queues[PACKET_DIR_INC];
+  int size_txq = queue_size[PACKET_DIR_OUT];
```

Comment:
Since only `size_txq` is updated in this function, queue_size[] values may not reflect the reality.

`int &size_txq = ...` might be a quick fix. I am not sure if having a reference of primitive variable is fine with the Google style guide, though.

Reply:
Yup, so will use queue_size directly, thanks.